frontend/simple/js/events.js

```diff
@@ -1,21 +1,37 @@
 import Vue from 'vue'
 import * as Events from '../../../shared/events'
-import store from './state'
 import backend from '../js/backend'
-import {namespace} from '../js/backend/hapi'
 import _ from 'lodash'
 
 export class GroupContract extends Events.HashableGroup {
   static vuex = GroupContract.Vuex({
-    state: { votes: [], payments: [], invitees: [], profiles: {} },
+    state: { votes: [], payments: [], invitees: [], profiles: {}, proposals: {} },
+    // NOTICE: All mutations are to be atomic in their edits of the contract state. THEY ARE NOT to farm out
+    // any further mutations through the async actions
     mutations: {
       GroupContract (state, {data}) {
-        Vue.set(state.profiles, data.founderUsername, {
-          contractId: store.state.loggedIn.identityContractId
-        })
+        Vue.set(state.profiles, data.founderUsername, {globalProfile: data.founderIdentityContractId, groupProfile: {}})
```

Comment:
Should probably rename `globalProfile` back to `contractId` as that's more accurate, and might help keep the rest of the code the same.